src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -43,6 +43,7 @@
  *  * NULL_ON_INVALID_REFERENCE:      Returns null
  *  * IGNORE_ON_INVALID_REFERENCE:    Ignores the wrapping command asking for the reference
  *                                    (for instance, ignore a setter if the service does not exist)
+ *  * IGNORE_ON_UNINITIALIZED_REFERENCE: Ignores/returns null for uninitialized services
```

Comment:
I would say `for uninitialized services or invalid references`, to be clear that it also includes the behavior of `IGNORE_ON_INVALID_REFERENCE`.

Btw, the phpdoc looks weird now, as the sentence before the bullet list says it impacts only cases where the service does not exist, which is not the case anymore. You may want to reword it a bit (btw, we should also rework the part about factory methods, as this talks about the deprecated API)

Reply:
docblock fixed